stable/traefik/README.md

```diff
@@ -87,12 +87,14 @@ The following tables lists the configurable parameters of the Traefik chart and
 | `acme.persistence.size`         | Minimum size of the volume requested                                 | `1Gi`                                     |
 | `dashboard.enabled`             | Whether to enable the Traefik dashboard                              | `false`                                   |
 | `dashboard.domain`              | Domain for the Traefik dashboard                                     | `traefik.example.com`                     |
-| `gzip.enabled`                  | Whether to use gzip compression                                      | `true`                     |
+| `dashboard.basicAuth`           | Basic auth for the Traefik dashboard, see Authentication section     | No auth                                   |
```

Comment:
Sorry... also... is there some way we can avoid possibly misleading anyone into thinking "no auth" is actually a default value? Can we use language like "unset by default; this means basic auth is disabled." No one will mistake that as the actual default value of the property.

Can you also add a section for this to the default values.yaml and leave it commented. It's easier for people to get this bit of config right if it's already present and commented; meaning they don't have to start from scratch and possibly get an indent wrong or something.